pandas-stubs/_libs/tslibs/timedeltas.pyi

```diff
@@ -130,9 +130,9 @@ class Timedelta(timedelta):
     @property
     def asm8(self) -> np.timedelta64: ...
     # TODO: round/floor/ceil could return NaT?
-    def round(self, freq: str | BaseOffset) -> Self: ...
-    def floor(self, freq: str | BaseOffset) -> Self: ...
-    def ceil(self, freq: str | BaseOffset) -> Self: ...
+    def round(self, freq: Frequency) -> Self: ...
+    def floor(self, freq: Frequency) -> Self: ...
+    def ceil(self, freq: Frequency) -> Self: ...
```

Comment:
docs say `str` but offset objects are also accepted, so I think it's ok to keep the current annotation
```python
In [11]: pd.Timedelta(minutes=37).round(pd.tseries.offsets.Hour())
Out[11]: Timedelta('0 days 01:00:00')
```